cinema/views.py

```diff
@@ -43,11 +47,57 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+
+        if title:
+            queryset = queryset.filter(title__icontains=title)
+
+        if genres:
+            genres_ids = [int(genre_id) for genre_id in genres.split(",")]
```

Comment:
Duplicate line(64) create separate method which will convert string to list with numbers